index.html

```diff
@@ -0,0 +1,159 @@
+<html>
+<head>
+	<title>Cognits</title>
+	<link rel="stylesheet" type="text/css" href="styles/bootstrap.min.css">
+	<link rel="stylesheet" type="text/css" href="styles/bootstrap-theme.min.css">
+	 <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
+	<link rel="stylesheet" type="text/css" href="styles/styles.css">
+</head>
+<body>
+<nav class="navbar navbar-inverse navbar-fixed-top ">
```

Comment:
El `.navbar` no esta exactamente del mismo color. Debe ser completamente negro, ahorita esta un negro con un poco de degrade, el cual es el que incluye Bootstrap de entrada. Para guia, pueden ver [este tutorial](http://www.dmxzone.com/support/22598/topic/59630/) o [esta respuesta en StackOverflow](http://stackoverflow.com/questions/24127521/change-bootstrap-navbar-background-color-and-font-color).

Vale la pena recordad que librerias y frameworks como Bootsrap son una herramienta que nos ayuda a hacer el trabajo mas rapido, pero es solo una herramienta. Casi siempre el trabajo va a requerir CSS.